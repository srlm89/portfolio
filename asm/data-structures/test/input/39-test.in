ERASE actual/39-test.out
INSERT 7 5.390656 The Matrix   1080p
PRINT_IN_ORDER actual/39-test.out
INSERT 38 0.740913 X-Men 1 - 1080p
INSERT 1 0.879318 Eurotrip 720p
REMOVE_ID 1
PRINT_IN_ORDER actual/39-test.out
INSERT 20 7.017877 Bourne Identity   720p
INSERT 21 2.147573 Kingdom of Heaven [2005] [720p]
INSERT 42 1.267029 Core   720p
REMOVE_ID 38
PRINT_IN_ORDER actual/39-test.out
INSERT 26 7.863319 Monsters vs Aliens - Mutant Pumpkins from Outer Space [2009] [720p]
INSERT 28 4.663409 Shark Tale [2004] [1080p]
INSERT 47 2.631031 Mission Impossible Trilogy   720p
INSERT 10 3.597300 Into The Blue 720P
REMOVE_ID 20
REMOVE_ID 10
PRINT_IN_ORDER actual/39-test.out
INSERT 11 3.382296 The Matrix Reloaded   1080p
INSERT 0 3.524694 Shoot.Em.Up.720p.Bluray.x264-SEPTiC
INSERT 3 4.691706 Madagascar 1 [2005] [1080p]
INSERT 15 7.119533 Gladiator 1080p
INSERT 16 5.017128 Up [1080p]
REMOVE_ID 3
REMOVE_ID 28
PRINT_IN_ORDER actual/39-test.out
INSERT 5 3.501340 Bride Wars   720p
INSERT 33 6.186802 American Gangster    720p
INSERT 46 0.934351 Doomsday 720p
INSERT 39 9.481517 Die Hard 1   720p
INSERT 49 4.741892 Rocky 2 [1979] [BluRay] [720p] [AC3]
INSERT 50 2.939986 Blade 3   720p
REMOVE_ID 26
REMOVE_ID 50
REMOVE_ID 49
PRINT_IN_ORDER actual/39-test.out
INSERT 51 6.628451 The Last Samurai [2004] [720p]
INSERT 4 6.034891 The.X.Files.Fight.The.Future.1998.720p.BluRay.x264-SiNNERS
INSERT 55 7.705490 007 Licence to Kill [1989] [BluRay] [720p] [DTS]
INSERT 24 1.954475 Yes Man   720p
INSERT 52 6.310173 Disturbia    720p
INSERT 62 5.481476 Spiderman 3   1080p
INSERT 22 2.785908 Babel.720p
REMOVE_ID 52
REMOVE_ID 62
REMOVE_ID 55
PRINT_IN_ORDER actual/39-test.out
INSERT 66 8.238115 La Lingerie [BluRay] [720p] [DTS]
INSERT 17 0.495878 Gran Torino [2008] [720p]
INSERT 64 7.125724 Juno 2007 720p
INSERT 67 7.797909 Die Hard 2   720p
INSERT 30 3.372732 Oceans.Eleven.2001.720p.HDDVD.x264-SiNNERS
INSERT 29 9.215926 Seven Swords    1080p
INSERT 43 5.901403 The Notebook [2004] [1080p]
INSERT 18 2.095022 Ongbak 2003 720p
REMOVE_ID 17
REMOVE_ID 18
REMOVE_ID 39
REMOVE_ID 33
PRINT_IN_ORDER actual/39-test.out
INSERT 25 8.665787 Land Of the Lost   720p
INSERT 70 1.376559 The.Escape.2009.1080p.BluRay.x264-BestHD
INSERT 63 5.092627 Braveheart 720p
INSERT 13 7.886786 The Messenger [720p]
INSERT 48 7.311171 007 Thunderball [1965]  [BluRay] [720p] [DTS]
INSERT 76 2.834223 Big Buck Bunny [1080p]
INSERT 54 5.739755 Back.to.the.Future.Part.II.1989.720p.HDTV.x264
INSERT 34 6.132637 Robots.WS.AC3.x264.BluRay-PTM   720p
INSERT 9 3.143814 Death Race   720p
REMOVE_ID 64
REMOVE_ID 51
REMOVE_ID 46
REMOVE_ID 22
PRINT_IN_ORDER actual/39-test.out
INSERT 27 5.161767 Alien vs Predator 1-2  720p
INSERT 61 1.925442 The Godfather [1972] [720p]
INSERT 2 2.989869 Banlieue 13 - Ultimatum [720p]
INSERT 74 8.274904 Pirates of Caribbean 1-2-3 Collection   1080p
INSERT 83 4.281458 CJ7   720p
INSERT 81 0.169189 Die Hard 2   720p
INSERT 57 6.120345 SAW I   720p
INSERT 80 2.449326 National Tressure 1 - 1080p
INSERT 91 5.828667 Ong.Bak.2.2008.BluRay.x264-BestHD   720p
INSERT 84 9.369704 Eurotrip    720p
REMOVE_ID 63
REMOVE_ID 30
REMOVE_ID 80
REMOVE_ID 48
REMOVE_ID 25
PRINT_IN_ORDER actual/39-test.out
INSERT 19 4.907904 Death Sentence   1080p
INSERT 92 4.653110 Inkheart [2008] [720p]
INSERT 68 3.047660 Night At The Museum 2 [720p]
INSERT 44 5.263512 Assassination of a High School President 720p
INSERT 93 1.929847 Con Air   720p
INSERT 45 6.663928 Robots.WS.AC3.x264.BluRay-PTM   720p
INSERT 23 6.354638 Gran Torino [2008] [720p]
INSERT 65 2.226397 Casablana.1942.1080p.BluRay.x264.Codres
INSERT 73 2.539974 Alien vs Predator 1-2  720p
INSERT 58 4.362843 The Matrix   1080p
INSERT 36 6.336966 Swordfish    720p
REMOVE_ID 83
REMOVE_ID 58
REMOVE_ID 44
REMOVE_ID 74
REMOVE_ID 68
PRINT_IN_ORDER actual/39-test.out
INSERT 82 1.315275 National Geographic - The Koran [720p] [Documenter]
INSERT 56 8.402227 Rambo 4 [2008] [720p]
INSERT 8 3.261763 Toy.Story.1995.720p.HDTV.x264-CtrlHD
INSERT 95 6.565982 Gone In 60 Seconds    1080p
INSERT 35 2.892460 contents.txt
INSERT 106 9.902546 Fearless - 720p
INSERT 85 4.168801 Blood River 720P
INSERT 37 1.423994 SAW I   720p
INSERT 99 1.100734 Darko 720P
INSERT 108 4.062322 Dark.knight, The.2008.720p.bluray.x264-sinners
INSERT 69 8.928866 Cloverfield   1080p
INSERT 88 2.844366 3_10 to Yuma   720p
REMOVE_ID 7
REMOVE_ID 91
REMOVE_ID 21
REMOVE_ID 8
REMOVE_ID 93
REMOVE_ID 56
PRINT_IN_ORDER actual/39-test.out
