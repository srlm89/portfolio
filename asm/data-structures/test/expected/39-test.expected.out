[ 7 5.390656 The Matrix   1080p ]
[ 7 5.390656 The Matrix   1080p ][ 38 0.740913 X-Men 1 - 1080p ]
[ 7 5.390656 The Matrix   1080p ][ 20 7.017877 Bourne Identity   720p ][ 21 2.147573 Kingdom of Heaven [2005] [720p] ][ 42 1.267029 Core   720p ]
[ 7 5.390656 The Matrix   1080p ][ 21 2.147573 Kingdom of Heaven [2005] [720p] ][ 26 7.863319 Monsters vs Aliens - Mutant Pumpkins from Outer Space [2009] [720p] ][ 28 4.663409 Shark Tale [2004] [1080p] ][ 42 1.267029 Core   720p ][ 47 2.631031 Mission Impossible Trilogy   720p ]
[ 0 3.524694 Shoot.Em.Up.720p.Bluray.x264-SEPTiC ][ 7 5.390656 The Matrix   1080p ][ 11 3.382296 The Matrix Reloaded   1080p ][ 15 7.119533 Gladiator 1080p ][ 16 5.017128 Up [1080p] ][ 21 2.147573 Kingdom of Heaven [2005] [720p] ][ 26 7.863319 Monsters vs Aliens - Mutant Pumpkins from Outer Space [2009] [720p] ][ 42 1.267029 Core   720p ][ 47 2.631031 Mission Impossible Trilogy   720p ]
[ 0 3.524694 Shoot.Em.Up.720p.Bluray.x264-SEPTiC ][ 5 3.501340 Bride Wars   720p ][ 7 5.390656 The Matrix   1080p ][ 11 3.382296 The Matrix Reloaded   1080p ][ 15 7.119533 Gladiator 1080p ][ 16 5.017128 Up [1080p] ][ 21 2.147573 Kingdom of Heaven [2005] [720p] ][ 33 6.186802 American Gangster    720p ][ 39 9.481517 Die Hard 1   720p ][ 42 1.267029 Core   720p ][ 46 0.934351 Doomsday 720p ][ 47 2.631031 Mission Impossible Trilogy   720p ]
[ 0 3.524694 Shoot.Em.Up.720p.Bluray.x264-SEPTiC ][ 4 6.034891 The.X.Files.Fight.The.Future.1998.720p.BluRay.x264-SiNNERS ][ 5 3.501340 Bride Wars   720p ][ 7 5.390656 The Matrix   1080p ][ 11 3.382296 The Matrix Reloaded   1080p ][ 15 7.119533 Gladiator 1080p ][ 16 5.017128 Up [1080p] ][ 21 2.147573 Kingdom of Heaven [2005] [720p] ][ 22 2.785908 Babel.720p ][ 24 1.954475 Yes Man   720p ][ 33 6.186802 American Gangster    720p ][ 39 9.481517 Die Hard 1   720p ][ 42 1.267029 Core   720p ][ 46 0.934351 Doomsday 720p ][ 47 2.631031 Mission Impossible Trilogy   720p ][ 51 6.628451 The Last Samurai [2004] [720p] ]
[ 0 3.524694 Shoot.Em.Up.720p.Bluray.x264-SEPTiC ][ 4 6.034891 The.X.Files.Fight.The.Future.1998.720p.BluRay.x264-SiNNERS ][ 5 3.501340 Bride Wars   720p ][ 7 5.390656 The Matrix   1080p ][ 11 3.382296 The Matrix Reloaded   1080p ][ 15 7.119533 Gladiator 1080p ][ 16 5.017128 Up [1080p] ][ 21 2.147573 Kingdom of Heaven [2005] [720p] ][ 22 2.785908 Babel.720p ][ 24 1.954475 Yes Man   720p ][ 29 9.215926 Seven Swords    1080p ][ 30 3.372732 Oceans.Eleven.2001.720p.HDDVD.x264-SiNNERS ][ 42 1.267029 Core   720p ][ 43 5.901403 The Notebook [2004] [1080p] ][ 46 0.934351 Doomsday 720p ][ 47 2.631031 Mission Impossible Trilogy   720p ][ 51 6.628451 The Last Samurai [2004] [720p] ][ 64 7.125724 Juno 2007 720p ][ 66 8.238115 La Lingerie [BluRay] [720p] [DTS] ][ 67 7.797909 Die Hard 2   720p ]
[ 0 3.524694 Shoot.Em.Up.720p.Bluray.x264-SEPTiC ][ 4 6.034891 The.X.Files.Fight.The.Future.1998.720p.BluRay.x264-SiNNERS ][ 5 3.501340 Bride Wars   720p ][ 7 5.390656 The Matrix   1080p ][ 9 3.143814 Death Race   720p ][ 11 3.382296 The Matrix Reloaded   1080p ][ 13 7.886786 The Messenger [720p] ][ 15 7.119533 Gladiator 1080p ][ 16 5.017128 Up [1080p] ][ 21 2.147573 Kingdom of Heaven [2005] [720p] ][ 24 1.954475 Yes Man   720p ][ 25 8.665787 Land Of the Lost   720p ][ 29 9.215926 Seven Swords    1080p ][ 30 3.372732 Oceans.Eleven.2001.720p.HDDVD.x264-SiNNERS ][ 34 6.132637 Robots.WS.AC3.x264.BluRay-PTM   720p ][ 42 1.267029 Core   720p ][ 43 5.901403 The Notebook [2004] [1080p] ][ 47 2.631031 Mission Impossible Trilogy   720p ][ 48 7.311171 007 Thunderball [1965]  [BluRay] [720p] [DTS] ][ 54 5.739755 Back.to.the.Future.Part.II.1989.720p.HDTV.x264 ][ 63 5.092627 Braveheart 720p ][ 66 8.238115 La Lingerie [BluRay] [720p] [DTS] ][ 67 7.797909 Die Hard 2   720p ][ 70 1.376559 The.Escape.2009.1080p.BluRay.x264-BestHD ][ 76 2.834223 Big Buck Bunny [1080p] ]
[ 0 3.524694 Shoot.Em.Up.720p.Bluray.x264-SEPTiC ][ 2 2.989869 Banlieue 13 - Ultimatum [720p] ][ 4 6.034891 The.X.Files.Fight.The.Future.1998.720p.BluRay.x264-SiNNERS ][ 5 3.501340 Bride Wars   720p ][ 7 5.390656 The Matrix   1080p ][ 9 3.143814 Death Race   720p ][ 11 3.382296 The Matrix Reloaded   1080p ][ 13 7.886786 The Messenger [720p] ][ 15 7.119533 Gladiator 1080p ][ 16 5.017128 Up [1080p] ][ 21 2.147573 Kingdom of Heaven [2005] [720p] ][ 24 1.954475 Yes Man   720p ][ 27 5.161767 Alien vs Predator 1-2  720p ][ 29 9.215926 Seven Swords    1080p ][ 34 6.132637 Robots.WS.AC3.x264.BluRay-PTM   720p ][ 42 1.267029 Core   720p ][ 43 5.901403 The Notebook [2004] [1080p] ][ 47 2.631031 Mission Impossible Trilogy   720p ][ 54 5.739755 Back.to.the.Future.Part.II.1989.720p.HDTV.x264 ][ 57 6.120345 SAW I   720p ][ 61 1.925442 The Godfather [1972] [720p] ][ 66 8.238115 La Lingerie [BluRay] [720p] [DTS] ][ 67 7.797909 Die Hard 2   720p ][ 70 1.376559 The.Escape.2009.1080p.BluRay.x264-BestHD ][ 74 8.274904 Pirates of Caribbean 1-2-3 Collection   1080p ][ 76 2.834223 Big Buck Bunny [1080p] ][ 81 0.169189 Die Hard 2   720p ][ 83 4.281458 CJ7   720p ][ 84 9.369704 Eurotrip    720p ][ 91 5.828667 Ong.Bak.2.2008.BluRay.x264-BestHD   720p ]
[ 0 3.524694 Shoot.Em.Up.720p.Bluray.x264-SEPTiC ][ 2 2.989869 Banlieue 13 - Ultimatum [720p] ][ 4 6.034891 The.X.Files.Fight.The.Future.1998.720p.BluRay.x264-SiNNERS ][ 5 3.501340 Bride Wars   720p ][ 7 5.390656 The Matrix   1080p ][ 9 3.143814 Death Race   720p ][ 11 3.382296 The Matrix Reloaded   1080p ][ 13 7.886786 The Messenger [720p] ][ 15 7.119533 Gladiator 1080p ][ 16 5.017128 Up [1080p] ][ 19 4.907904 Death Sentence   1080p ][ 21 2.147573 Kingdom of Heaven [2005] [720p] ][ 23 6.354638 Gran Torino [2008] [720p] ][ 24 1.954475 Yes Man   720p ][ 27 5.161767 Alien vs Predator 1-2  720p ][ 29 9.215926 Seven Swords    1080p ][ 34 6.132637 Robots.WS.AC3.x264.BluRay-PTM   720p ][ 36 6.336966 Swordfish    720p ][ 42 1.267029 Core   720p ][ 43 5.901403 The Notebook [2004] [1080p] ][ 45 6.663928 Robots.WS.AC3.x264.BluRay-PTM   720p ][ 47 2.631031 Mission Impossible Trilogy   720p ][ 54 5.739755 Back.to.the.Future.Part.II.1989.720p.HDTV.x264 ][ 57 6.120345 SAW I   720p ][ 61 1.925442 The Godfather [1972] [720p] ][ 65 2.226397 Casablana.1942.1080p.BluRay.x264.Codres ][ 66 8.238115 La Lingerie [BluRay] [720p] [DTS] ][ 67 7.797909 Die Hard 2   720p ][ 70 1.376559 The.Escape.2009.1080p.BluRay.x264-BestHD ][ 73 2.539974 Alien vs Predator 1-2  720p ][ 76 2.834223 Big Buck Bunny [1080p] ][ 81 0.169189 Die Hard 2   720p ][ 84 9.369704 Eurotrip    720p ][ 91 5.828667 Ong.Bak.2.2008.BluRay.x264-BestHD   720p ][ 92 4.653110 Inkheart [2008] [720p] ][ 93 1.929847 Con Air   720p ]
[ 0 3.524694 Shoot.Em.Up.720p.Bluray.x264-SEPTiC ][ 2 2.989869 Banlieue 13 - Ultimatum [720p] ][ 4 6.034891 The.X.Files.Fight.The.Future.1998.720p.BluRay.x264-SiNNERS ][ 5 3.501340 Bride Wars   720p ][ 9 3.143814 Death Race   720p ][ 11 3.382296 The Matrix Reloaded   1080p ][ 13 7.886786 The Messenger [720p] ][ 15 7.119533 Gladiator 1080p ][ 16 5.017128 Up [1080p] ][ 19 4.907904 Death Sentence   1080p ][ 23 6.354638 Gran Torino [2008] [720p] ][ 24 1.954475 Yes Man   720p ][ 27 5.161767 Alien vs Predator 1-2  720p ][ 29 9.215926 Seven Swords    1080p ][ 34 6.132637 Robots.WS.AC3.x264.BluRay-PTM   720p ][ 35 2.892460 contents.txt ][ 36 6.336966 Swordfish    720p ][ 37 1.423994 SAW I   720p ][ 42 1.267029 Core   720p ][ 43 5.901403 The Notebook [2004] [1080p] ][ 45 6.663928 Robots.WS.AC3.x264.BluRay-PTM   720p ][ 47 2.631031 Mission Impossible Trilogy   720p ][ 54 5.739755 Back.to.the.Future.Part.II.1989.720p.HDTV.x264 ][ 57 6.120345 SAW I   720p ][ 61 1.925442 The Godfather [1972] [720p] ][ 65 2.226397 Casablana.1942.1080p.BluRay.x264.Codres ][ 66 8.238115 La Lingerie [BluRay] [720p] [DTS] ][ 67 7.797909 Die Hard 2   720p ][ 69 8.928866 Cloverfield   1080p ][ 70 1.376559 The.Escape.2009.1080p.BluRay.x264-BestHD ][ 73 2.539974 Alien vs Predator 1-2  720p ][ 76 2.834223 Big Buck Bunny [1080p] ][ 81 0.169189 Die Hard 2   720p ][ 82 1.315275 National Geographic - The Koran [720p] [Documenter] ][ 84 9.369704 Eurotrip    720p ][ 85 4.168801 Blood River 720P ][ 88 2.844366 3_10 to Yuma   720p ][ 92 4.653110 Inkheart [2008] [720p] ][ 95 6.565982 Gone In 60 Seconds    1080p ][ 99 1.100734 Darko 720P ][ 106 9.902546 Fearless - 720p ][ 108 4.062322 Dark.knight, The.2008.720p.bluray.x264-sinners ]
